src/main/java/pl/gov/coi/pomocua/ads/captcha/CaptchaException.java

```diff
@@ -0,0 +1,13 @@
+package pl.gov.coi.pomocua.ads.captcha;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import pl.gov.coi.pomocua.ads.error.CodeException;
+import pl.gov.coi.pomocua.ads.error.CodeValidationError;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class CaptchaException extends CodeException {
+    public CaptchaException(CodeValidationError validationError) {
```

Comment:
If `CaptchaException` is thrown just from a single place, maybe it could have a no-args constructor and create `CodeValidationError` here?

Reply:
Done